playbooks/luna_demo_environment.yml

```diff
@@ -0,0 +1,80 @@
+# Manually verify your hosts has FQDN set correctly
+# check it's in /etc/hosts /etc/hostname and hostname returns it, /etc/hosts first line should be 192.168.73.1  $fqdn
+
+# TODO - add role for setting up the machine - autologin in gdm, perhaps background image etc, installation of virt-manager, vim, htop
+# TODO - refactor katello_provisioning so that it does not run installer, it should only seed stuff, installer should be customized from playbook using foreman_installer's role variables
+
+- hosts: all
+  become: true
+  roles:
+    - disable_ipv6
+    - disable_firewall
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - ostree_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - foreman_installer
+    - libvirt_setup
+    - katello_provisioning
+    - plugins/foreman_templates
+# openscap manual steps
+# 1. edit Base/Katello CentOS 7/OpenSCAP host group and set the policy manually
+# covered by http://projects.theforeman.org/issues/22481
+    - plugins/foreman_openscap
+# discovery manual steps (to make discovery work via pxelinux):
+# 1. unlock PXELinux global default
+# 2. change ONTIMEOUT local to ONTIMEOUT discovery
+# 3. save
+# 4. click Build PXE Default button
+# covered by http://projects.theforeman.org/issues/21204/
```

Comment:
This one is closed so is it still an issue?

Reply:
as I mentioned above, the discovery setup could probably be improved already, I don't have cycles to fiddle with settings through hammer now